### PR TITLE
Include badge in facet link.

### PR DESCRIPTION
**Pull request by EreMaijala**

### CODE REVIEW

**Comment by demiankatz** [comment]
I ran tests and they're all passing for me. Functionality also looks good. But I'll wait to see if @ThoWagen approves before taking further action.

File: themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml [hunk 1]
```diff
@@ -34,31 +34,24 @@
...
-  ob_start();
-?>
-<?php if (!$this->facet['isApplied'] && null !== $this->facet['count']): ?>
-    <span class="badge">
-      <?php $currentCount = $this->localizedNumber($this->facet['count']); ?>
-      <span class="sr-only"><?=$this->transEsc('result_count', ['%%count%%' => $currentCount]) ?></span>
-      <span aria-hidden="true"><?=$currentCount ?></span>
+<span class="<?=implode(' ', $classList) ?>">
+  <a class="text<?=$hasCheckbox ? ' icon-link' : ''?>" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if ($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
+    <span class="text">
```
Comment (ThoWagen) [inline themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml:39]: Having the class "text" on both the link and this span seems a bit counterintuitive to me.

But with the current styles we need them in both cases and it works. Maybe that is something we can revise in the future.

**Review by ThoWagen — APPROVED** [review]
Besides the small comment below this looks good to me. Thanks @EreMaijala!

File: themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml (hunk 1 above)
Comment (EreMaijala) [reply themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml:39]: True, but I decided to keep it because it's back-compatible.

Comment (EreMaijala) [reply themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml:39]: And yes, we could perhaps revisit the styles in a major release.

**Review by demiankatz — APPROVED** [review]
Thanks, @EreMaijala and @ThoWagen! I'll merge this as-is, and we can revisit in the dev branch if we want to make bigger changes.

**Pull request merged**